src/main/kotlin/com/programmersbox/fullmultiplatformcompose/BuilderParams.kt

```diff
@@ -10,7 +10,7 @@ class BuilderParams {
     var hasiOS: Boolean by mutableStateOf(false)
     var hasDesktop: Boolean by mutableStateOf(false)
 
-    var packageName by mutableStateOf("com.example")
+    var packageName by mutableStateOf("idv.neo.compose")
```

Comment:
Can you please change this back to `com.example`?

Reply:
sorry

I sent the wrong PR

I mainly want to change the versions of various libs (especially the kotlin part, which I want to upgrade to 2.0.20 or above)

I also added my commonly used repos and libs, and then sent the PR without distinguishing them.

Mainly I want to know if this plugin can be modified to support kotlin 2.0 or above and more lib and mvvm templates?

Reply:
It definitely can. I just don't have the time to work on it plus there's the official web wizard which works well